src/Neo.CLI/CLI/MainService.cs

```diff
@@ -356,6 +360,8 @@ public async void Start(string[] args)
                         break;
                 }
 
+            await TryInstallLevelDbPlugin();
```

Comment:
It's asking if I want to install LevelDB even if I'm using Memory Store

Reply:
fixed.